### PR TITLE
Refactors menu tab

### PR DESCRIPTION
**Pull request by SuchAFuriousDeath**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,25 +1,22 @@
...
-  ],
-  "rust-analyzer.cargo.features": "all",
-  "rust-analyzer.imports.granularity.group": "module",
-  "rust-analyzer.imports.group.enable": false,
-  "rust-analyzer.imports.prefix": "self"
+	"[rust]": {
+		"editor.formatOnSave": true,
+		"editor.rulers": [100]
+	},
+	"slint.preview.style": "fluent-dark",
```
[the remark this answers is not in the document]
Comment (SuchAFuriousDeath) [reply .vscode/settings.json:10]: We probably should have put it in the settings.json as well a long time ago :sweat_smile:

[the remark this answers is not in the document]
Comment (SuchAFuriousDeath) [reply .vscode/settings.json:10]: Wait, how is the diff still different?

[the remark this answers is not in the document]
Comment (ultimaweapon) [reply .vscode/settings.json:10]: Seems like to changed to tab instead of spaces.

File: gui/slint/main/menu.slint [hunk 2]
```diff
@@ -62,117 +84,64 @@ export component Menu {
...
-                Rectangle {
-                    Text {
-                        text: "File";
+                MenuItem {
+                    title: "File";
+
+                    Button {
+                        text: "Install PKG";
+                        clicked => { install_pkg(); }
                     }
```
Comment (ultimaweapon) [inline gui/slint/main/menu.slint:110]: We don't need this anymore.

**Review by ultimaweapon — CHANGES_REQUESTED** [review]

**Pull request merged**